lrpar/cttests/src/calc_parse_param.test

```diff
@@ -0,0 +1,45 @@
+name: Test %parse-param using the calculator grammar
+yacckind: Original(YaccOriginalActionKind::UserAction)
+grammar: |
+    %start Expr
+    %actiontype Result<u64, ()>
+    %avoid_insert 'INT'
+    %parse_param <'a>(x : &'a mut u64)
+    %%
+    Expr: Expr '+' Term {
+                          *x = $1? + $3?;
```

Comment:
I think the indentation would match the other `.test` files if it was more like:

```rust
Expr: Expr '+' Term {
        *x = $1? + $3?;
        ...
    }
```

I think the `Factor` rule's indent precedent is probably worth reusing?

Reply:
I believe I got this matching the existing tests.

Reply:
I think the indentation later in the same file is different, so it's probably worth syncing it, at least within that single file.

Reply:
I think now I get what you mean.